OpenRA.Game/FileFormats/PngLoader.cs

```diff
@@ -111,6 +111,9 @@ public static Bitmap Load(Stream s)
 
 								case "IEND":
 									{
+										if (bitmap == null)
+											throw new InvalidDataException("Image header not found.");
```

Comment:
There is a check [a bit further up](https://github.com/OpenRA/OpenRA/pull/10905/files#diff-d982fdbe9f03fe9111a8f46963a257abR55) for this. Either I've messed it up and we should fix that check, or the static analysis isn't clever enough to realise it can't get to here with a null bitmap. We should try and fix that check rather than inserting redundant code.